### ISSUE TITLE
Anyone else having this issue?

### ISSUE THREAD

**Issue by hclcrome** [finndev/PokeBuddy#811]
#700 This dose anyone suffer from the same thing?

**Comment by drdetroit1** [comment]
Yes. Ive tried every solution i could find in this chat with no success. Only thing i can think of is im downloading a old zip or something. But i can not find a different one to download .......

**Comment by curtiola** [comment]
An old zip would just auto update anyway, but I'm sorry I don't know the solution to the problem
